### PR TITLE
develop: add proxysql integration

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add a document about how to integrate TiDB Cloud with ProxySQL. [preview](https://add-develop-proxysql.website-docs.pages.dev/tidbcloud/dev-guide-proxysql-integration)

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-proxysql-integration.md [hunk 1]
```diff
@@ -86,6 +108,14 @@ The production environment requires more steps than the test environment. To dep
 
 </SimpleTab>
 
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
+
+It is recommended to use [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) directly when you need hosting TiDB services (for example, you cannot manage it yourself, or you need a cloud-native environment). To build a TiDB cluster in a production environment, refer to [Create a TiDB cluster](https://docs.pingcap.com/tidbcloud/create-tidb-cluster).
```
Comment (qiancai) [inline develop/dev-guide-proxysql-integration.md:115]:
```suggestion
To build a TiDB cluster in a production environment, refer to [Create a TiDB cluster](https://docs.pingcap.com/tidbcloud/create-tidb-cluster).
```

File: develop/dev-guide-proxysql-integration.md [hunk 2]
```diff
@@ -5,9 +5,22 @@ summary: Introduce how to integrate TiDB with ProxySQL step by step.
...
 
 - [TiDB Documentation](/overview.md)
+
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
+
+This document describes how to integrate **TiDB Cloud** with **ProxySQL** using CentOS 7 as an example. If you want to integrate using other systems, refer to the [Try Out](#4-try-out) section, which introduces how to deploy a test integration environment using **Docker** and **Docker Compose**. For more information, refer to:
+
+- [TiDB Cloud Documentation](/tidb-cloud/tidb-cloud-intro.md)
```
Comment (qiancai) [inline develop/dev-guide-proxysql-integration.md:20]:
```suggestion
This document describes how to integrate **TiDB Cloud** with **ProxySQL** using CentOS 7 as an example. If you want to integrate using other systems, refer to the [Try Out](#4-try-out) section, which introduces how to deploy a test integration environment using **Docker** and **Docker Compose**. 
```

Comment (qiancai) [reply develop/dev-guide-proxysql-integration.md:20]: When users are viewing this doc, they are already opening the Cloud docs website, so there is no need to direct them to the Cloud introduction again.

File: develop/dev-guide-proxysql-integration.md [hunk 3]
```diff
@@ -86,6 +111,14 @@ The production environment requires more steps than the test environment. To dep
 
 </SimpleTab>
 
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
+
+To build a TiDB cluster in a production environment, refer to [Create a TiDB cluster](https://docs.pingcap.com/tidbcloud/create-tidb-cluster).
```
Comment (qiancai) [inline develop/dev-guide-proxysql-integration.md:118]:
```suggestion
To build a TiDB cluster in a production environment, refer to [Create a TiDB cluster](/tidb-cloud/create-tidb-cluster.md).
```

**Review by Icemap — APPROVED** [review]

**Pull request merged**